backend/spec/system/question_crud_spec.rb

```diff
@@ -22,14 +22,23 @@
           find('.option', text: question_type.name).click
 
           find('label', text: 'User').click
+        end
+
+        it 'creates the question for admin' do
           find('.option', text: user_admin.email).click
 
           fill_in 'Name', with: 'question name'
 
           click_button 'Create Question'
+          expect(page).to have_content 'Question was successfully created.'
         end
 
-        it 'creates the question' do
+        it 'creates the question for user' do
+          find('.option', text: user_teacher.email).click
```

Comment:
should validate that user select is not available for user that is not admin

Reply:
Ok, I will add those tests as well! Thanks for your feedback.